docs/CSBR.md

```diff
@@ -1374,7 +1376,7 @@ The CA SHALL protect its Private Key in a system or device that has been validat
 
 #### 6.2.7.2 Private key storage for Timestamp Authorities
 
-A Timestamp Authority MUST protect its signing key using a process that is at least to FIPS 140-2 level 3, Common Criteria EAL 4+ (ALC\_FLR.2), or higher. The CA MUST protect its signing operations in accordance with the CA/Browser Forum's Network and Certificate System Security Requirements.
+Effective April 15, 2025, a Timestamp Authority MUST protect Private Keys associated with its Root CA certificates and Subordinate CA certificates containing the "Time Stamping" EKU in offline Hardware Crypto Module conforming to FIPS 140-2 level 3, Common Criteria EAL 4+ (ALC_FLR.2), or higher. The Timestamp Authority MUST protect its signing operations in accordance with the CA/Browser Forum's Network and Certificate System Security Requirements.

```

Comment:
I think it's preferable to re-use existing requirements for Root keys here, as that will provide better clarity on the expectations. Specifically, I'm concerned that the term "offline" is not specific.

What do you think about stating that the Timestamp CA Private Key must be stored in a "Root CA System" (a defined term in the NSRs)?

Reply:
I agree that there's a potential for misalignment due to the defined term explicitly speaking to Root key material, whereas we're dealing with Timestamping SubCA key material here.

I think your amended language is spot on, but needs a "a" before "Hardware Crypto Module". I'd suggest the edit myself, but apparently you can't suggest changes to suggestions (preventing Suggestion Inception, or something).

Reply:
Thanks for spotting the a, as well as the other suggestions! I've updated the previous suggestion and will commit.